code/game/turfs/turf.dm

```diff
@@ -533,6 +533,9 @@ GLOBAL_LIST_INIT(unweedable_areas, typecacheof(list(
 			else
 				has_obstacle = TRUE
 				break
+		if(istype(O, /obj/effect/alien/weeds/node/strong/hivemindcore))
```

Comment:
I'm on the opinion that we should up the status of weed nodes in general and not allow constructions over them.